app/models/application.rb

```diff
@@ -82,4 +84,8 @@ def project1
   def project2
     Project.find_by(id: application_data['project2_id'])
   end
+
+  def remove_duplicate_flags
```

Comment:
Since we're not using this outside of the scope of the `Application` model, this should definitely be in the `private` section 👮

Reply:
My bad! Makes sense, will do.